src/pages/OnboardingAccounting/BaseOnboardingAccounting.tsx

```diff
@@ -81,14 +80,9 @@ function BaseOnboardingAccounting({shouldUseNativeStyles, route}: BaseOnboarding
                         src={accountingIcon}
                         width={variables.iconSizeExtraLarge}
                         height={variables.iconSizeExtraLarge}
-                        additionalStyles={[
-                            StyleUtils.getAvatarBorderStyle(CONST.AVATAR_SIZE.DEFAULT, CONST.ICON_TYPE_AVATAR),
-                            styles.mr3,
-                            onboardingIsMediumOrLargerScreenWidth ? styles.ml3 : styles.ml0,
-                        ]}
+                        additionalStyles={[StyleUtils.getAvatarBorderStyle(CONST.AVATAR_SIZE.DEFAULT, CONST.ICON_TYPE_AVATAR), styles.mr3]}
```

Comment:
Do we still need `styles.mr3`?

Reply:
Yes we still need `mr3` to add the space between the icon and the text.